### PR TITLE
created new pom class

### PR DESCRIPTION
**Pull request by barsharani28**
please review

### CODE REVIEW

File: ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectRepositryutility/CreateProductPage.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import com.comcast.crm.generic.webdriverutility.WebDriverUtility;
+
+public class CreateNewOrganizationPage {
+
+	public static void main(String[] args) {
+		//code
+	}
```
Comment (Barsha28) [inline ComcastCRMGUIFramework/src/main/java/com/comcast/crm/objectRepositryutility/CreateProductPage.java:14]: don't hard code the data
change the absolute xpath to relative xpath

**Review by Barsha28 — COMMENTED** [review]
added few comments

**Comment by barsharani28** [comment]
fixed

**Comment by Barsha28** [comment]
merged

**Pull request merged**